includes/Models/DataStore/VendorOrderStatsStore.php

```
@@ -240,4 +245,230 @@ protected function fill_missing_dates( array $data, string $start_date, string $
+    }
+
+    /**
+     * Get report summary from dokan_order_stats table.
+     *
+     * @since DOKAN_SINCE
+     *
+     * @param array $args Query arguments.
+     *
+     * @return array Summary totals.
+     */
+    public function get_report_summary( array $args = [] ): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)